tests/damage_rheology_strain/statistics

```diff
@@ -0,0 +1,24 @@
+# 1: Time step number
+# 2: Time (years)
+# 3: Time step size (years)
+# 4: Number of mesh cells
+# 5: Number of Stokes degrees of freedom
+# 6: Number of temperature degrees of freedom
+# 7: Number of degrees of freedom for all compositions
+# 8: Iterations for temperature solver
+# 9: Iterations for composition solver 1
+# 10: Iterations for Stokes solver
+# 11: Velocity iterations in Stokes preconditioner
+# 12: Schur complement iterations in Stokes preconditioner
+# 13: Minimal value for composition olivine_grain_size
+# 14: Maximal value for composition olivine_grain_size
+# 15: Global mass for composition olivine_grain_size
+# 16: Minimal temperature (K)
+# 17: Average temperature (K)
+# 18: Maximal temperature (K)
+# 19: Average nondimensional temperature (K)
+# 20: RMS velocity (m/year)
+# 21: Max. velocity (m/year)
+0 0.000000000000e+00 0.000000000000e+00 256 2467 1089 1089 0  0 30 31 92 1.00000000e-03 1.00000010e-03 1.00000005e+07 1.60000000e+03 1.60000000e+03 1.60000000e+03 3.73686192e-15 5.77350322e-01 9.92956151e-01 
+1 3.125000000000e+03 3.125000000000e+03 256 2467 1089 1089 0 14  0  0  0 9.98808671e-04 9.98808791e-04 9.98808733e+06 1.60000000e+03 1.60000000e+03 1.60000000e+03 3.73686192e-15 5.77350322e-01 9.92956151e-01 
+2 5.000000000000e+03 1.875000000000e+03 256 2467 1089 1089 0  6  0  0  0 9.98106927e-04 9.98107057e-04 9.98106995e+06 1.60000000e+03 1.60000000e+03 1.60000000e+03 3.73686192e-15 5.77350322e-01 9.92956151e-01 
```

Comment:
The original results for the end time are: 9.98106935e-04 9.98107043e-04 9.98106992e+06 (for min, max, average grainsize). So those are very similar, I would say this test is working.